### PR TITLE
Clarify that implementations can use stable prefixes once an MSC has merged

### LINKED ISSUE #3146: Update proposal process: Allow stable usage after FCP concluded

### PR DESCRIPTION
**Pull request by anoadragon453**
Fixes #3146.

This PR changes the Matrix Spec Proposals page to clarify that implementations **do not** need to wait until a spec release to use stable prefixes, but that they can do so after the corresponding MSC has been merged. The justification is that once an MSC has been merged, it's fairly guaranteed that it will land in the spec. Yet it will take time for the spec release process to run its course, and we shouldn't make implementations wait for that.

The exception to this is if implementating a feature in a backwards-compatible way requires a new spec version to indicate to clients/servers that a feature has been added/changed. This situation is rare though, and most implementations won't fall into this category.

There's a couple changes at the top of the diff which were just correcting errors in the document.

### CODE REVIEW

File: content/proposals.md [hunk 1]
```diff
@@ -429,22 +427,25 @@ that a particular MSC works) do not have to follow this process.
...
     MSC lands.
-6.  A spec PR is written to incorporate the changes into Matrix.
-7.  A spec release happens.
-8.  Implementations switch to using stable prefixes (e.g.: `/r0`) if the
-    server supports the specification version released. If the server
-    doesn't advertise the specification version, but does have the
-    feature flag, unstable prefixes should still be used.
-9.  A transition period of about 2 months starts immediately after the
+1.  Implementations can now switch to using stable prefixes
+    (e.g.: for an endpoint, moving from
```
Comment (richvdh) [inline content/proposals.md:440]: .: is a bit unwieldy.

```suggestion
    (for example, for an endpoint, moving from
```

File: content/proposals.md [hunk 2]
```diff
@@ -463,9 +464,9 @@ com.example/new/endpoint`.
 
 In summary:
 
--   Implementations MUST NOT use stable endpoints before the MSC is in
-    the spec. This includes NOT using stable endpoints in the period
-    between completion of FCP and release of the spec. passed.
+-   Implementations MUST NOT use stable endpoints before the MSC has
+    been merged. Once an MSC has been accepted and merged, implementations
```
Comment (richvdh) [inline content/proposals.md:468]: according to the above, it's not neccesary for an MSC to be merged to the spec. Can we relax this?

**Review by richvdh — CHANGES_REQUESTED** [review]

File: content/proposals.md (hunk 2 above)
Comment (anoadragon453) [reply content/proposals.md:468]: Ah, by "accepted and merged" I meant just that the MSC proposal document needs to be merged - not that the spec PR needs to be merged.

I'll try to clarify.

File: content/proposals.md [hunk 3]
```diff
@@ -463,9 +464,10 @@ com.example/new/endpoint`.
 
 In summary:
 
--   Implementations MUST NOT use stable endpoints before the MSC is in
-    the spec. This includes NOT using stable endpoints in the period
-    between completion of FCP and release of the spec. passed.
+-   Implementations MUST NOT use stable endpoints before the MSC proposal
+    document has been merged to the matrix-doc repository. Once that has
```
[the remark this answers is not in the document]
Comment (anoadragon453) [reply content/proposals.md:468]: Sure, that is less wordy.

**Pull request merged**